dips/dip-158.md

```diff
@@ -0,0 +1,795 @@
+Successful authorization will not necessarily be followed by a successful capture. It is possible that the receiver will choose to cancel the authorization and essentially void the payment before it was captured. One possible reason to void a payment is that the products for which an amount was locked are out of stock and cannot be fulfilled. 
+
+---
+## New Elements and Existing Elements Extensions
+---
+This section specifies extensions for existing elements (e.g. new values) and introduces new ones required for P2M scenarios.
+
+
+## PaymentActorObject Extension
+The PaymentActorObject is extended to include the field `merchant_data` of type `MerchantDataObject`.
+Field|Type|Required?|Description|
+|-|-|-|-|
+|merchant_data|MerchantDataObject|N|This element is required for some scenarios. It includes the details of the merchant to be displayed to the payer|
+|
+
+## PaymentActor Status Enum Extension
+The following values are added to the status enum of [PaymentActor](https://dip.diem.com/dip-1/#paymentactorobject) object:
+
+* `authorized` - This means the actor has performed all required tasks and funds are locked for a future capture. Note that the actual lock is done by the wallet (sender), so for the receiver this status means that all risk checks were successful and it is willing to accept the payment. 
+* `needs_init_data` - This means the actor requires basic data to initialize the payment. For the receiver, this status means that the payer data is required. For the sender, it means that the merchant data is required  
+
+For completeness purposes, the full list of values is: 
+* `none`
+* `needs_kyc_data`
+* `ready_for_settlement`
+* `abort`
+* `authorized`
+* `soft_match`
+* `needs_init_data`
+
+Allowed transition relevant for P2M:
+
+| Current Status | Next Status | Transition Comments |
+|-|-|-|
+| none | needs_init_data |Initial transition. Most common for P2M scenarios in which the actor requires additional data to initialize the payment|
+| needs_init_data | ready_for_settlement | All required tasks were performed and the payment is ready for settlement |
+| needs_init_data | authorized | All required tasks were performed and the funds are ready to be locked |
+| authorized | ready_for_settlement | Funds captured |
+| authorized | abort | Payment was voided |
+| needs_init_data | abort | Payment aborted before initial data was verified |
+| none | abort | Payment aborted before initialization |
+| none | ready_for_settlement | All required tasks were performed and the payment is ready for settlement. It may happen when no payer data is required |
+| none | authorized |  All required tasks were performed and the funds are ready to be locked. It may happen when no payer data is required |
+| ready_for_settlement | abort | One party aborted the payment even though the other party already notified it is ready to settle the payment. For example, it may happen if the sender does not approve merchant data provided by the receiver |
+|
```

Comment:
delete

Reply:
Deleted the two transitions from none to ready_for_settlement/authorized